preprocessing/filtered_innerouter_snsc_mousekidney.ipynb

```diff
@@ -210,90 +210,6 @@
     "bulk_adata = bulk_adata.transpose()"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    plt.title('MSE of Bulk Expr. vs. Average Expr. in SN and SC (filtered)')
</pre><p>can you show this after taking the log of the MSE? Its hard to see the distribution when there are so many large outliers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/rnaseq_translator/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='128'/>